pkgs/applications/altcoins/lnd.nix

```diff
@@ -0,0 +1,23 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  name = "lnd-${version}";
+  version = "0.7.0-beta";
+
+  src = fetchFromGitHub {
+    owner = "lightningnetwork";
+    repo = "lnd";
+    rev = "v${version}";
+    sha256 = "0d6m1vfy33rg6d7qmkpydiypav1girxsnxan9njyjz0vhinmq0sx";
+  };
+
+  modSha256 = "0akxi7xhyz7xx0vc003abidva02sp940cc2gfjg4fmzkc95cajc9";
+
+  meta = with lib; {
+    description = "Lightning Network Daemon";
+    homepage = https://github.com/lightningnetwork/lnd;
+    license = lib.licenses.mit;
+    maintainers = with maintainers; [ cypherpunk2140 ];
+    platforms = platforms.unix;
```

Comment:
Is this necessary? `buildGoModule` defines it to platforms.all if found undefined.

Reply:
I was under the impression `lnd` doesn't work on Windows, but I was wrong, good catch.